Allow language modules to be installed in the right place.

For Python, ensure the use of the vendor-packages directory, as well as the
use of a 64-bit-specific directory, as needed.

This is an in-house Solaris specific patch not intended for upstream.

--- graphviz-9.0.0/configure.ac
+++ graphviz-9.0.0/configure.ac
@@ -1088,7 +1088,7 @@ else
 	           use_python3="No (python-$PYTHON3_VERSION.pc not found)"
           fi
         fi
-        PYTHON3_INSTALL_DIR="`$PYTHON3 -c 'import sysconfig; print(sysconfig.get_path("platlib"))'`"
+        PYTHON3_INSTALL_DIR="/usr/lib/python${PYTHON3_VERSION}/vendor-packages"
         save_CPPFLAGS=$CPPFLAGS
         CPPFLAGS="$CPPFLAGS $PYTHON3_INCLUDES"
         AC_CHECK_HEADER(Python.h,,[
